Sources/Cheers.swift

```diff
@@ -16,7 +16,7 @@ public class CheerView: UIView {
     stop()
 
     let emitter = CAEmitterLayer()
-    emitter.emitterPosition = CGPoint(x: bounds.width / 2.0, y: 0)
+    emitter.emitterPosition = CGPoint(x: UIScreen.main.bounds.width / 2.0, y: 0)
```

Comment:
We should prefer using `bounds` instead of `UIScreen`. Can you check that you have the correct `frame size` by the time you call `start`? Also, does it happen with the demo project?